code/script.js

```diff
@@ -1,51 +1,215 @@
+    // the else if statement checks if the sender is a bot and if that's the case it inserts an html section inside the chat with the posted message
   } else if (sender === 'bot') {
+    console.log ("The bot is writing")
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/botemoji.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
-  }
+  
+}
+
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
 }
 
+//This function makes the questions and answers appear in the correct order. 
+
+const nextLine = (message) => {
+  console.log ('lineNumber', lineNumber)
+
+ if (lineNumber === 1) {
+    userReply (message) 
+    userInput.value = ''    
+    setTimeout(() => secondLine (message), 1000)
+  } else if (lineNumber === 2) {
+    userReply(message)
+    userInput.value = ''    
+    setTimeout(() => thirdLine (message), 1000) ; 
+  } else if (lineNumber === 3) {
+    userReply(message)
+    userInput.value = ''    
+    setTimeout(() => fourthLine (message), 1000) ; 
+  } else if (lineNumber === 4) {
+    userReply(message)
+    userInput.value = '' 
+    setTimeout(() => fifthLine (message), 1000) ; 
+     } else if (lineNumber === 5) {
+    userReply(message)
+    userInput.value = '' 
+    setTimeout(() => sixthLine (message), 1000) ; 
+  } else if (lineNumber === 6) {
+    userReply(message)
+    userInput.value = ''    
+    setTimeout(() => seventhLine (message), 1000) ; 
+  } else if (lineNumber === 7) {
+    userReply(message)
+    userInput.value = ''    
+    setTimeout(() => eighthLine (message), 1000) ; 
+  } else if (lineNumber === 8) {
+    userReply(message)
+    userInput.value = ''    
+    setTimeout(() => ninthLine (message), 1000) ; 
+  } else {
+    userReply (message)
+    userInput.value=""
+    setTimeout(() => tenthLine (message), 1000) ; 
+
+}; 
+}
+
 // Starts here
-const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+
+
+//First reply from bot 
+const firstLine = () => {
+  lineNumber = 1; 
+  botReply(`Knock, knock.`);
 }
 
+//Second reply from bot
+const secondLine = () => {
+lineNumber++;  
+botReply(`Tank.`);
+} 
+
+//Third reply from bot
+  const thirdLine = () => {
+  lineNumber++
+  botReply(`You're welcome!`);
+  setTimeout(() => botReply (`Knock, knock.`), 2000) ; 
+  }
+
+  //Fourth reply from bot 
+  const fourthLine = () => {
+    lineNumber++;  
+    botReply(`Boo.`);
+  }
```

Comment:
In this section the indentation varies a bit. You start with two functions that have no indentation, but on the following functions after that there are indentations. Maybe take a look? In my mind there is no need to intend the functions, since they are on their own. Do you agree?